hllplus/hllplus_private_test.go

```diff
@@ -0,0 +1,50 @@
+package hllplus
+
+import (
+	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/gomega"
+)
+
+var _ = Describe("sparseState", func() {
+	var subject *sparseState
+
+	BeforeEach(func() {
+		subject = newSparseState(12, 17, nil)
+	})
+
+	It("should handle duplicates properly", func() {
+		subject.Add(1)
+		subject.Flush()
+		Expect(subject.data.Count()).To(BeNumerically("==", 1))
+
+		subject.Add(1)
+		subject.Flush()
+		Expect(subject.data.Count()).To(BeNumerically("==", 1))
+
+		subject.Add(1)
+		subject.Add(2)
+		subject.Add(1)
+		subject.Flush()
+		Expect(subject.data.Count()).To(BeNumerically("==", 2))
+	})
+})
+
+var _ = Describe("sparseState", func() {
```

Comment:
deltaSlice you mean

Reply:
actually, I don't like us to test private structs. they are in-fact just fine, there is no need to de-dupe anything in deltaSlice, pls remove this file again